os-datavirt/added/launch/teiid.sh

```diff
@@ -16,6 +16,9 @@ function prepareEnv() {
   unset DATAVIRT_USERS
   unset DATAVIRT_USER_PASSWORDS
   unset DATAVIRT_USER_GROUPS
+  unset JDBC_SECURITY_DOMAIN
+  unset ODBC_SECURITY_DOMAIN
+  unset ODATA_SECURITY_DOMAIN
```

Comment:
Why are we defining separate domains for each transport?  Is this a common use case? In the event we do need this, would it be good to have a DEFAULT_SECURITY_DOMAIN that gets applied to all transports?

Reply:
Understood, but the ability to specify a common default should make things easier for most use cases, e.g.
```
DEFAULT_SECURITY_DOMAIN=${DEFAULT_SECURITY_DOMAIN:-teiid-security}
JDBC_SECURITY_DOMAIN=${JDBC_SECURITY_DOMAIN:-${DEFAULT_SECURITY_DOMAIN}}
...
```